host/libvirt_lxc_backend.go

```diff
@@ -76,6 +76,7 @@ type LibvirtLXCBackend struct {
 	state     *State
 	pinkerton *pinkerton.Context
 
+	ifaceMTU   int
```

Comment:
At some point we should pass it to containerinit, the other option is not parsing the MTU out of the file for now.